magefile.go

```diff
@@ -102,6 +102,11 @@ func Lint() error {
 		return err
 	}
 
+	fmt.Println("Running luacheck for test projects files...")
+	if err := sh.RunV(".rocks/bin/luacheck", "test/files"); err != nil {
```

Comment:
What error does user see if there is not luacheck installed?

Reply:
The same as errors where `python3` or `go` isn't installed.